docs/proposals/L7-traffic-route.md

```diff
@@ -0,0 +1,174 @@
+# L7 Traffic Routing
+
+## Context
+
+Kuma provides traffic management with `TrafficRoute` policy based on Kuma tags.
+We can split the traffic to different destinations, we can redirect the traffic etc.
+
+## Requirements
+
+Provide a way to manage the traffic based on HTTP headers, path and method.
+
+## Configuration
+
+```yaml
+type: TrafficRoute
+name: route-all-default
+mesh: default
+sources:
+- match:
+    kuma.io/service: web
+destinations:
+- match:
+    kuma.io/service: backend
+conf:
+  http: # a new http section which is a list
+  - match: # match is required and need to have at least one sub-element
+      method: GET
+      path: # one of either prefix, exact or regex will be allowed
+        prefix: /users
```

Comment:
Are `/users/foo` and `/users`, `/users/, and `/usersfoo` all prefix matches in this example?

Reply:
correct

Reply:
Ok, then suggest we clarify that in the spec here :)